go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/Azure/azure-sdk-for-go/sdk/keyvault/azsecrets v0.12.0
 	github.com/Masterminds/semver v1.5.0
 	github.com/Microsoft/go-winio v0.6.1
-	github.com/Microsoft/hcsshim v0.11.4
+	github.com/Microsoft/hcsshim v0.12.0-rc.2
```

Comment:
What caused this update?

Reply:
we need to fetch this hcsshim version for:
hcnNetwork.Flags = hcn.DisableHostPort

Reply:
Considering it is a release candidate update. Can you also run the CNI Release Test Pipeline? Just want to ensure there are not additional changes that are breaking.

Reply:
we cannot merge with release candidate version.. have to get official version released

Reply:
This is in rc status as they are making 0.12.X dependent on containerd 2.0, which has not been released yet.